common/src/API.php

```diff
@@ -2,6 +2,10 @@
 namespace Valued\WordPress;
 
 use Exception;
+
+if (!defined('REQUESTS_SILENCE_PSR0_DEPRECATIONS')) {
+    define('REQUESTS_SILENCE_PSR0_DEPRECATIONS', true);
+}
 use Requests;
```

Comment:
`use` statements don't actually load anything, they just import a certain namespace.  So you can keep the `use` statements at the top as usual.

```suggestion
use Requests;

if (!defined('REQUESTS_SILENCE_PSR0_DEPRECATIONS')) {
    define('REQUESTS_SILENCE_PSR0_DEPRECATIONS', true);
}
```